UI/src/view/speedLimitWorkList.js

```diff
@@ -0,0 +1,109 @@
+(function (root) {
+  root.SpeedLimitWorkList = function(){
+    MunicipalityWorkList.call(this);
+    var me = this;
+    this.hrefDir = "#work-list/speedLimit/municipality";
+    var municipalityList;
+    var showFormBtnVisible = true;
+
+    this.bindEvents = function () {
+      eventbus.on('speedLimitMunicipality:select', function(listP, stateHistory) {
+        $('.container').hide();
+        $('#work-list').show();
+        $('body').addClass('scrollable');
+        municipalityList = listP;
+        me.generateWorkList(listP, stateHistory);
+      });
+    };
+
+    this.createVerificationForm = function(municipality) {
+      $('#tableData').hide();
+      $('.filter-box').hide();
+      if (showFormBtnVisible)
+        $('#work-list-header').append($('<a class="header-link"></a>').attr('href', me.hrefDir).html('Kuntavalinta').click(function(){
+          me.generateWorkList(municipalityList);
+        }));
+      me.reloadForm(municipality);
+    };
+
+    this.municipalityTable = function (municipalities, filter) {
+      var municipalityValues =
+        _.isEmpty(filter) ? municipalities : _.filter(municipalities, function (municipality) {
+          return municipality.name.toLowerCase().startsWith(filter.toLowerCase());
+        });
+
+      var tableContentRows = function (municipalities) {
+        return _.map(municipalities, function (municipality) {
+          return $('<tr/>').append($('<td/>').append(idLink(municipality)));
+        });
+      };
+      var idLink = function (municipality) {
+        return $('<a class="work-list-item"/>').attr('href', me.hrefDir/* + '/' + municipality.id*/).html(municipality.name).click(function(){
+          me.createVerificationForm(municipality);
+        });
+      };
+      return $('<table id="tableData"/>').append(tableContentRows(municipalityValues));
+    };
+
+    this.workListItemTable = function(layerName, workListItems, municipalityName) {
+      var municipalityHeader = function(municipalityName, totalCount) {
+        var countString = totalCount && layerName !== 'speedLimit' ? ' (yhteensä ' + totalCount + ' kpl)' : '';
```

Comment:
should be refactor, because we are on speed limit work list, so we don't need the layername validation